packages/orbit-components/src/Stack/consts.js

```diff
@@ -25,12 +25,12 @@ export const JUSTIFY = {
 
 export const SPACINGS = {
   NONE: "none",
-  EXTRATIGHT: "extraTight",
-  TIGHT: "tight",
-  CONDENSED: "condensed",
-  COMPACT: "compact",
-  NATURAL: "natural",
-  COMFY: "comfy",
-  LOOSE: "loose",
-  EXTRALOOSE: "extraLoose",
+  XXXSMALL: "XXXSmall",
+  XXSMALL: "XXSmall",
+  XSMALL: "XSmall",
+  SMALL: "small",
+  MEDIUM: "medium",
+  LARGE: "large",
+  XLARGE: "XLarge",
+  XXLARGE: "XXLarge",
 };
```

Comment:
We didn't support it in the past, because `52px` was just abstracted from old designs. I don't know about any real use case for it and we were talking in the past about dropping this spacing completely 🙂